### PR TITLE
Icy lake tiles and pits

### PR DESCRIPTION
**Pull request by chandhana9**
Description:
Added the tiles for the icy lake under tilemaps. Edited code for stop watch to load game over screen at 60 seconds. Edited code in player Manager so when character touches a pit, game over screen appears. Did not place the pits in the game yet, instead made it a prefab so it can be placed wherever needed. 

Image:
Screenshot 2024-01-24 210636

## Checklist:
- [X] Title
- [X] Description of work
- [X] Images or videos
- [X] Notified team of PR

### CODE REVIEW

File: src/UBC Toboggan/Assets/Scripts/playerManager.cs [hunk 1]
```diff
@@ -146,22 +146,34 @@ void Update()
...
-        if (playerTrigger.IsTouching(collider) && collider.tag == "ground") {
+        if (playerTrigger.IsTouching(collider) && collider.tag == "ground")
+        {
+            alive = false;
+            fire.SetFloat("isBoosting", 0f);
+            boostAmount = 0f;
+            StartCoroutine(LoadGameOverScreen());
+        }
+        if (skiTrigger.IsTouching(collider) && collider.tag == "pit")
+        {
```
Comment (dominiccupidon) [inline src/UBC Toboggan/Assets/Scripts/playerManager.cs]: These two if statements result in the same action, so let's combine them into one statement that says 
```cs
if ((playerTrigger.isTouching(collider) && collider.tag == "ground") 
    || (skiTrigger.isTouching(collider) && collider.tag == "pit")) {
     alive = false;
     fire.SetFloat("isBoosting", 0f);
     boostAmount = 0f;
     StartCoroutine(LoadGameOverScreen());
}
```

File: src/UBC Toboggan/Assets/Scripts/playerManager.cs [hunk 2]
```diff
@@ -146,22 +146,34 @@ void Update()
...
             alive = false;
             fire.SetFloat("isBoosting", 0f);
             boostAmount = 0f;
             StartCoroutine(LoadGameOverScreen());
         }
     }
+
+    //
```
Comment (dominiccupidon) [inline src/UBC Toboggan/Assets/Scripts/playerManager.cs]: Is this a missing comment?

File: src/UBC Toboggan/Assets/Scripts/StopWatch.cs [hunk 3]
```diff
@@ -21,10 +25,11 @@ void Update()
         if (isTimerRunning)
         {
             secondsElapsed += Time.deltaTime;
-            if (secondsElapsed > 90) // Consider setting the limit to 7.5 minutes (450 seconds)
+            if (secondsElapsed > 60) // Consider setting the limit to 7.5 minutes (450 seconds)
```
Comment (dominiccupidon) [inline src/UBC Toboggan/Assets/Scripts/StopWatch.cs]: Please remove the comment beside this if statement since we'll no longer be doing that

**Comment by AidanCotsakis** [comment]
I have a few comments about the art,
- There seems to be some missing pixels throughout and some pixels that are not fully opaque which would allow the background to be visible through the ground.
- There seems to be some single pixels extending out of the 16x16 tile which will make making a tilemap much more challenging
- Some of the slopes have particularly flat sections near the edges which may make making a tilemap harder.
- There is a border on the bottom of the icy slopes meaning you will not be able to add a blank tile under them seamlessly.
- There are no tiles for an icy slope going the other way (the ice slopes need an inverted variant)
- I cannot seem to find the aseprite file in the commit.

**Review by dominiccupidon — APPROVED** [review]

**Review by AidanCotsakis — APPROVED** [review]

**Pull request merged**